data/src/main/java/com/my/kizzy/data/rpc/RpcImage.kt

```diff
@@ -87,4 +87,21 @@ sealed class RpcImage {
             }
         }
     }
+
+    class URLImage(val url: String) : RpcImage() {
```

Comment:
It's redundant, `URLImage` is basically `ExternalImage`, so use it instead. Although I would suggest since the images are static upload them to some discord application assets and use that asset url(which never expires). And now we can have a simple `DiscordImage`

Reply:
Right I forgot app assets are a thing lol, will do later

Reply:
> It's redundant, `URLImage` is basically `ExternalImage`, so use it instead. 

`ExternalImage` does **not** cache images

Reply:
Here use them, they are uploaded under Kizzy app

play: https://cdn.discordapp.com/app-assets/962990036020756480/1300361266212241430.png?size=512

pause: https://cdn.discordapp.com/app-assets/962990036020756480/1300361619490209802.png?size=512

stop: https://cdn.discordapp.com/app-assets/962990036020756480/1300361702621188160.png?size=512